runtime/commonTest/src/kotlinx/serialization/json/JsonMapKeysTest.kt

```diff
@@ -37,19 +37,19 @@ class JsonMapKeysTest : JsonTestBase() {
                 streaming
             )
         }
-        assertTrue(e.message?.contains("can't be used in json as map key") == true, "wrong exception text")
+        assertTrue(e.message?.contains("can't be used in JSON as a key in the map") == true)
```

Comment:
Why you've erased assertion message?

Reply:
Because it creates visual noise and is meaningless anyway